AIDriver.lua

```diff
@@ -525,7 +525,7 @@ function AIDriver:onEndCourse()
 			courseplay:stop(self.vehicle)
 			-- TODO: encapsulate this in an AutoDriveInterface class
 			local parkDestination = self.vehicle.spec_autodrive:GetParkDestination(self.vehicle)
-			self.vehicle.spec_autodrive:StartDrivingWithPathFinder(self.vehicle, parkDestination, 0, nil, nil, nil)
+			self.vehicle.spec_autodrive:StartDrivingWithPathFinder(self.vehicle, parkDestination, -3, nil, nil, nil)
```

Comment:
Can we use constants (defined by AD I guess) instead of magic numbers?

Reply:
I don´t like them neither, but is the only way I found that doesn´t break game if not updating the other mod. First I was going to use diferent functions but this would break backward compatibility. In fact the "magic number" was used before, but changed to -3 for parking and -2 for refuel. -1 is AD default and AD can use all positive numbers as destinations. If you know a way to improve it please do it. I'm just a learner trying to do my best, but I have no programing knowledge compared to yours. 
Thank you.

Reply:
Got it. Thanks for the great contribution.